README.md

```diff
@@ -1,23 +1,23 @@
-# Futbol
+## Futbol
 
 Starter repository for the [Turing School](https://turing.io/) Futbol project.
 
 
-A 1-2 summary or bullet points outlining your plan for check-ins throughout the duration of the project.
+# - A 1-2 summary or bullet points outlining your plan for check-ins throughout the duration of the project.
```

Comment:
Do you want to drop these `<h1>`s to like `<h2>` or `<h3>` just so they aren't so... large? 👀  Open to ideas or feedback from others too on it